### PR TITLE
enable html keys, tested

### PR DESCRIPTION
**Pull request by ohadpartuck**
enable for users to send html keys for special cases where colors and bold are needed.

### CODE REVIEW

**Comment by yesmeck** [comment]
@ohadpartuck Thanks, but this project is written in CoffeScript, you should edit  files under `src/` directory, not in `dist/`

File: dist/jquery.jsonview.js [hunk 1]
```diff
@@ -250,11 +252,13 @@ Licensed under the MIT License.
       defaultOptions = {
         collapsed: false,
         nl2br: false,
-        recursive_collapser: false
+        recursive_collapser: false,
+        enableHTMLKeys: false
```
Comment (yesmeck) [inline dist/jquery.jsonview.js]: I'd like to keep options names simple, mayby just call this option `escape` and set it's default to true.

Comment (ohadpartuck) [inline dist/jquery.jsonview.js]: done.

**Comment by ohadpartuck** [comment]
@yesmeck edited the coffescript. lmk if it's ok +  should I cancel my changes at `dist/` all together?

**Comment by yesmeck** [comment]
@ohadpartuck Just run `gulp dist`, files under dist will generate automatically.

File: src/jquery.jsonview.coffee [hunk 2]
```diff
@@ -45,10 +45,12 @@ do (jQuery) ->
         collapsed: false,
         nl2br: false,
         recursive_collapser: false
+        escape: false
```
Comment (yesmeck) [inline src/jquery.jsonview.coffee]: `escape` should default enable to prevent XSS attack.

**Comment by ohadpartuck** [comment]
Had to install all the dev dependencies https://github.com/ohadpartuck/jquery-jsonview/blob/master/package.json#L22
ran the `gulp dist`. getting an error . evan on master. any chance you can `fetch` and test your self (I don't really have a coffee script state of mind right now)
screen shot 2015-11-09 at 9 51 57 am

**Comment by yesmeck** [comment]
Ok, just revert the changes of `dist`, I'll handle it.

**Comment by ohadpartuck** [comment]
10x, done.

**Comment by yesmeck** [comment]
:+1:

**Pull request merged**